app/_data/docs_nav_ee_0.35-x.yml

```diff
@@ -183,7 +183,7 @@
       - text: Session Plugin
         url: /plugins/session
       - text: Route By Headers
-        url: /plugins/route-by-headers
+        url: /plugins/route-by-header
```

Comment:
@levfishbluefish this commit already changed it back to singular; staging may be outdated.

Reply:
Disregard!